src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/SelectStandardOption.cshtml

```diff
@@ -68,5 +69,20 @@
 
 @section breadcrumb
 {
-    <a id="back-link" class="govuk-back-link" href="@Url.Action("Details", "Cohort", new { Model.ProviderId, Model.CohortReference })">Back</a>
-}
+    @if (Model.HasSelectedRpl == true)
```

Comment:
This is ok, but I wonder if we could move the logic into the model mapper itself to allow for unit testing?